### DISCUSSION TITLE
Is it possible to clone a website for unblocking?

### DISCUSSION THREAD

**Discussion by Charlieis1nsan3** [3kh0/ext-remover#1228] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Is it possible to create a clone of a game site? I want the actual game to run in html not just a index.html that doesn't run anything could somebody help? if possible??????

**Comment by akabutnicer** [comment, marked as answer]
You can't copy the server side of a game site in HTML unless it happens to be internal. For instance, take the repository: `@scheng123321/1v1-lol/blob/master/index.html`.
The game uses Firebase which is an external backend/database.
